include/cantera/zeroD/Reactor.h

```diff
@@ -93,6 +93,17 @@ class Reactor : public ReactorBase
      */
     virtual void getState(doublereal* y);
 
+    //! Get the current state and derivative vector of the reactor for a DAE solver
+    /*!
+     *  @param[out] y     state vector representing the initial state of the reactor
+     *  @param[out] ydot  state vector representing the initial derivatives of the
+     *                    reactor
+     */
+    virtual void getState(doublereal* y, doublereal* ydot)
+    {
+        throw CanteraError("Reactor::getState", "Not Implemented");
```

Comment:
To differentiate from the other `getState` method:
```suggestion
        throw CanteraError("Reactor::getState(y, ydot)", "Not Implemented");
```